app/components/trainees/record_actions/view.rb

```diff
@@ -22,6 +22,36 @@ def allow_defer?
       def allow_withdraw?
         allow_defer? || trainee.deferred?
       end
+
+      def defer_and_withdraw_links
+        if choose_both_actions?
+          defer_path + " or " + withdraw_path
+        elsif allow_defer?
+          defer_path
+        elsif allow_withdraw?
+          reinstate_path + " or " + withdraw_path
+        else
+          ""
+        end
+      end
+
+    private
+
+      def choose_both_actions?
+        allow_defer? && allow_withdraw?
+      end
+
+      def defer_path
```

Comment:
These return links so they should be `defer_link`, `withdraw_link`, `reinstate_link`

Reply:
fixed